src/plugins/charts/public/services/colors/color_palette.ts

```diff
@@ -65,8 +65,8 @@ export function createColorPalette(num: number): string[] {
     throw new TypeError('ColorPaletteUtilService expects a number');
   }
 
-  const colors = seedColors;
-  const seedLength = seedColors.length;
+  const colors = euiPaletteColorBlind();
+  const seedLength = colors.length;
 
   _.times(num - seedLength, function (i) {
     colors.push(hsl((fraction(i + seedLength + 1) * 360 + offset) % 360, 50, 50).hex());
```

Comment:
let's remove this in favor just making sure we get enough colors from `euiPaletteColorBlind()` so we can just return a slice of those directly without any additional transformation.

Reply:
oh I see! Sorry I thought we will do any modification to this function as part of #4320. Ya I think we can get rid of this transformation logic and just use rotations on oui palette to fill the array. For now I've used 10 rotations, total of 100 colors which seems to be sufficient.